audit/src/repo/audit.ts

```diff
@@ -4,29 +4,13 @@ import { InfraLogger as logger } from '../logger';
 import * as Knex from 'knex';
 
 export class KnexAuditRepository implements AuditRepository {
-  private TABLE_NAME = 'audit';
-
-  public constructor(private readonly knex: Knex<{}, unknown[]>) {
-    this.initTables();
-  }
-
-  private async initTables(): Promise<void> {
-    // this will eventually be replaced with migrations
-
-    await this.knex.schema.createTable(this.TABLE_NAME, (table) => {
-      table.string('id');
-      table.string('subject');
-      table.string('verb');
-      table.string('entity');
-    });
-
-  }
+  public constructor(private readonly knex: Knex<{}, unknown[]>) { }
 
   public async putLog( item: AuditLogItem ): Promise<boolean> {
 
     logger.debug('auditWritten', item.id);
 
-    await this.knex(this.TABLE_NAME).insert<AuditLogItem>(item);
+    await this.knex('audit').insert<AuditLogItem>(item);
```

Comment:
I'd prefer we have a local list of tables e.g. `this.AUDIT_TABLENAME`

Reply:
where do we put it though ? the migrations would need to have access to the constant

Reply:
Just local to the class... I expect there will be other statements of `this.knex('audit')` so would be useful to just have one place to change it in the AuditRepository class

Reply:
the problem would be that the migration file would need to import the audit repo... not sure if thats a good idea.

Maybe a separate file with constants?

Reply:
i'll add it as a separate issue

Reply:
see #404